indico/htdocs/sass/modules/_markdown.scss

```diff
@@ -0,0 +1,301 @@
+/* This file is part of Indico.
```

Comment:
apparently at least one of the CSS rules is (currently) used when displaying an abstract (`.md-preview-wrapper.display`). No idea if we can get rid of it in the new abstracts module but we could keep the current name just in case and rename it later if we end up using it really only for editing. Would be a bit weird  to require CSS from a `_editor` module on display pages (even though it'll probably be included on all abstracts-related pages anyway)